subsys/usb/device_next/class/usbd_uvc.c

```diff
@@ -1624,6 +1642,7 @@ static int uvc_init(struct usbd_class_data *const c_data)
 	struct video_caps caps;
 	uint32_t prev_pixfmt = 0;
 	uint32_t mask = 0;
+	int first_err = ret;
```

Comment:
Compile error, that does not work in C. `int first_err = 0;` ?

Reply:
Adjusted in latest git push, sorry.